server/sonar-server/src/main/java/org/sonar/server/measure/ws/ComponentTreeData.java

```diff
@@ -140,4 +140,35 @@ public ComponentTreeData build() {
       return new ComponentTreeData(this);
     }
   }
+
+  static class Measure {
+    private Double value;
+    private String data;
+    private Double variation;
```

Comment:
Instead of two fields `Double`, you should use `double` with value `Double.NaN` when field is supposed to be null.